gigamon/logs/Gigamon.yml

```diff
@@ -0,0 +1,128 @@
+id: gigamon
```

Comment:
Can you rename this file to be `gigamon.yaml` and the tests file to be `gigamon_tests.yaml`? I don't think these are being processed via our linter at the moment.

Reply:
I see this comment was resolved but don't see the changes?